bazel/python_swagger.bzl

```diff
@@ -0,0 +1,113 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains parts derived from [1] and [2].
+# [1] Licensed under Apache License 2.0
+#     Copyright 2019 OpenAPI-Generator-Bazel Contributors
+#     https://github.com/OpenAPITools/openapi-generator-bazel/blob/fb7e302de4597277bea12757836f2ce988c805ee/internal/openapi_generator.bzl
+# [2] Licensed under MIT License
+#     Copyright (c) 2021 Meetup, Inc.
+#     https://github.com/meetup/rules_openapi/blob/86fa11d0a8a8188ceecb74b5674af3f7363701e8/openapi/openapi.bzl
+# We thank the authors of [1] and [2].
+
+""" Provides support for building python code from swagger specifications. """
+
+load("@bazel_tools//tools/build_defs/repo:jvm.bzl", "jvm_maven_import_external")
+
+SWAGGER_CLI_VERSION = "2.4.16"
+SWAGGER_CLI_SHA256 = "154b5a37254a3021a8cb669a1c57af78b45bb97e89e0425e3f055b1c79f74a93"
```

Comment:
This might be mentioned somewhere but where do you get the sha value from?

Reply:
This was copied from the sources mentioned in the file. But we verified it: 
`$ sha256sum swagger-codegen-cli-2.4.16.jar 
154b5a37254a3021a8cb669a1c57af78b45bb97e89e0425e3f055b1c79f74a93  swagger-codegen-cli-2.4.16.jar`
Too bad, maven central does not provide sha256. Easy way via bazel (but security net is kind of circumvented): remove argument "artifact_sha256", do bazel build, find output 
`INFO: SHA256 (https://repo.maven.apache.org/maven2/io/swagger/swagger-codegen-cli/2.4.16/swagger-codegen-cli-2.4.16.jar) = 154b5a37254a3021a8cb669a1c57af78b45bb97e89e0425e3f055b1c79f74a93`

Reply:
I think I will forget this information ^ at some point. Can we add it in as a comment? :)

Reply:
Added comment.